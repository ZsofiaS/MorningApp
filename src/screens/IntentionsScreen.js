import React from 'react';
import { View, Text } from 'react-native';

export default function IntentionsScreen() {
  return(
    <View>
      <Text> Set your intentions for today</Text>
    </View>
  )
}
