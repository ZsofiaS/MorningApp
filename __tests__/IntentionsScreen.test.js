import React from 'react';
import renderer from 'react-test-renderer';
import IntentionsScreen from '../src/screens/IntentionsScreen';

test('intentionsscreen renders correctly', () => {
  const tree = renderer.create(<IntentionsScreen />).toJSON();
  expect(tree).toMatchSnapshot();
})
