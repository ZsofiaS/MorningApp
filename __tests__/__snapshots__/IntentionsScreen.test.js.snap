// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`homescreen renders correctly 1`] = `
<View>
  <Text>
     Set your intentions for today
  </Text>
</View>
`;

exports[`intentionsscreen renders correctly 1`] = `
<View>
  <Text>
     Set your intentions for today
  </Text>
</View>
`;
